content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,206 @@ Calendar
+  will be a third option: *Select Time then auto-assign*.
+
+Schedule
+~~~~~~~~
+
+In the :guilabel:`Schedule` tab of the new meeting page, add or delete time slots.
+
+.. image:: calendar/calendar-new-meeting.png
+   :align: center
+   :alt: New meeting page on Calendar app.
+
+Options
+~~~~~~~
+
+The :guilabel:`Options` tab provides additional configurations:
+
+- :guilabel:`Front-End Display`: Select **Show Pictures** to share users' or resources' photos on
+  the linked meeting page.
+
+- :guilabel:`Timezone`: This is defaulted to the company's time zone selected in **Configuration**.
+  To change the time zone, click into the section and select from the drop-down menu.
+
+- :guilabel:`Location`: Select or create new locations on the drop-down menu.
+
+- :guilabel:`Videoconference Link`: Select from Odoo Discuss or Google Meet to include a video
+  conference link in the meeting invitation.
+
+- :guilabel:`Up-front Payment`: Tick the checkbox to require users to pay before confirming their
+  booking. Once this is checked, a link will appear to :guilabel:`Configure Payment Providers`,
+  which will enable online payments.
+
+- :guilabel:`Limit to Work Hours`: Tick the checkbox to limit meeting time slots to
+  :doc:`users' working hours <../hr/employees/new_employee>`.
+
+- :guilabel:`Create Opportunities`: When this is selected, each scheduled appointment will create
+  a new CRM opportunity.
+
+- :guilabel:`Reminders`: Add or delete notification reminders. Select the blank space for additional
+  options.
+
+- :guilabel:`Confirmation Email`: Tick the checkbox to automatically send a confirmation email to
+  attendees once the meeting is confirmed. Select from the email templates or click
+  :guilabel:`Search More` to create a custom template.
+
+- :guilabel:`Cancellation Email`: Tick the checkbox to automatically send a cancellation email to
+  attendees if the meeting is canceled. Select from the email templates or click
+  :guilabel:`Search More` to create a custom template.
+
+- :guilabel:`CC to`: Add contacts to be notified of meeting updates, regardless if they attend
+  the meeting.
+
+- :guilabel:`Allow Guests`: Tick the checkbox to allow invitees to invite guests.
+
+Questions
+~~~~~~~~~
+
+In the questions tab, add questions for the invitee to answer when confirming their meeting.
+Questions can be configured for question type, placeholder answers, and whether they are required.
+
+Messages
+~~~~~~~~
+
+In the :guilabel:`Introduction Message` section, add additional meeting information that will appear
+on the invitation.
+
+Information added to the :guilabel:`Extra Message on Confirmation` section will appear once the
+meeting is confirmed.
+
+Click the :guilabel:`Preview` button to see how the appointment link will look for attendees.
+
+Once the configuration is finished, select :guilabel:`Share` to generate a link to send directly, or
+:guilabel:`Publish` to publish the appointment selection on the connected Odoo website.
```

Comment:
At the moment, this information is appearing in the 'Messages' tab section. However, since it's a more generic note about the meeting form, in general, it might be better served prior to the breaking down of each tab...

```suggestion
Click the :guilabel:`Preview` button to see how the appointment link looks for attendees.

Once the configurations are finished, click the :guilabel:`Share` button to generate a link to send directly, or click
:guilabel:`Publish` to publish the appointment selection on the connected Odoo website.
```